src/smart-components/workflow/add-stages/add-stages-wizard.js

```diff
@@ -25,7 +25,7 @@ const AddWorkflow = ({
 
   const steps = [
     { name: 'General Information', component: <StageInformation formData={ formData } handleChange={ handleChange } /> },
-    { name: 'Set Stages', component: <SetStages formData={ formData }
+    { name: 'Set Groups', component: <SetStages formData={ formData }
```

Comment:
We should use sentence case `Set groups`. Can you fix it here or do you want to do these in separate PR?

Reply:
I'll fix it here.